index.js

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env node
-var pkg = require('./package.json');
-var Main$foreign = require('./output/Main/foreign.js');
-var Main = require('./output/Main');
-Main$foreign.version = pkg.version;
+import pkg from './package.json' assert { type: 'json' };
+import * as Main from './dist/index.js';
+global.version = pkg.version;
```

Comment:
I don't think that there's a need to set a global variable here. We could have `main :: String -> Effect Unit` that takes the version as an argument.

Alternatively, we could bake in the version number like I did in purs-tidy https://github.com/natefaubion/purescript-tidy/blob/33910b845a5b72952c43084890b651fa657bd1ce/package.json#L17-L18 then we wouldn't need to load the package.json at all.